Lib/configparser.py

```diff
@@ -877,7 +883,7 @@ def has_option(self, section, option):
         """Check for the existence of a given option in a given section.
         If the specified `section' is None or an empty string, DEFAULT is
         assumed. If the specified `section' does not exist, returns False."""
-        if not section or section == self.default_section:
+        if (not section and section != '') or section == self.default_section:
```

Comment:
It was stated in [issue22253](http://bugs.python.org/issue22253) that an empty string `''` is probably the best way to go. But now seems that It breaks backwards compatibility and it's against PEP 387.